Project-I-Social-Card/social-card/src/App.js

```diff
@@ -1,25 +1,25 @@
 import React from 'react';
-import './App.css';
+import ReactDOM from 'react-dom';
+
 
-import Header from './components/HeaderComponents';
+import Header from './components/HeaderComponents/HeaderContainer';
+import HeaderContent from './components/HeaderComponents/HeaderContent';
+import CardContent from './components/CardComponents/CardContent'
```

Comment:
I do notice that there are a few components that you are importing but not using in this file